_benchmark/user-guide/understanding-workloads/choosing-a-workload.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Choosing a workload
+nav_order: 20
+grand_parent: User guide
+parent: Understanding workloads
+---
+
+# Choosing a workload
+
+The [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) repository contains a list of workloads that you can use to run your benchmarks. Using a workload similar to your cluster's use cases can save you time and effort when assessing your cluster's performance. 
+
+For example, say you're a system architect at a ride share company. As a ride share company you collect and store data based on trip times, locations, and other data related to each ride share. Instead of building a custom workload and using your own data, which takes additional time, effort, and cost, you can use the [nyc_taxis](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/nyc_taxis) workload to benchmark your cluster, because the data inside the workload is similar to the data you collect. 
```

Comment:
This is a new one for me, but most of the references I see online use "rideshare" as one word, so let's follow suit. Also, for Global English purposes, let's use a more precise word than "takes," which can have more than one meaning. Additionally, as with "as well as," you almost never need a comma before "because."